utils/kata-manager.sh

```diff
@@ -669,16 +669,26 @@ handle_kata()
 	kata-runtime --version
 }
 
+containerd_version_number()
+{
+	local flavour="${1:-lts}"
```

Comment:
I think it would be better to set the default value of "lts" in `handle_args()` since:

- That avoids having to set the default twice - here and in `validate_containerd_flavour()`.
- I think the code would be clearer since flavour would always have a value rather than the current "magic empty string" behaviour.

Reply:
I totally agree, I've addressed that!